platform/platform-scheduler/src/main/java/ddf/platform/scheduler/ScheduledCommandTask.java

```diff
@@ -106,53 +138,85 @@ public void deleteTask(int code) {
     }
 
     @Override
-    public void updateTask(Map properties) {
-
-        if (properties == null) {
-            LOGGER.info("No properties detected. No action taken.");
+    public void updateTask(Map<String, Object> properties) {
+        if (MapUtils.isEmpty(properties)) {
+            LOGGER.info("Empty or null properties map. No action taken.");
             return;
         }
-        Object commandValue = properties.get(CommandJob.COMMAND_KEY);
-
-        Object intervalValue = properties.get("intervalInSeconds");
 
+        Object commandValue = properties.get(CommandJob.COMMAND_KEY);
         if (commandValue != null) {
             this.command = commandValue.toString();
         }
 
-        if (intervalValue != null) {
-            this.intervalInSeconds = Integer.parseInt(intervalValue.toString());
+        Object intervalString = properties.get(INTERVAL_STRING);
+        if (intervalString != null) {
+            LOGGER.debug("Updating intervalString : {}", intervalString);
+            this.intervalString = (String) intervalString;
         }
 
-        JobDetail newJob = createJob();
+        Object intervalType = properties.get(INTERVAL_TYPE);
+        if (intervalType != null) {
+            LOGGER.debug("Updating intervalType : {}", intervalType);
+            this.intervalType = (String) intervalType;
+        }
 
+        JobDetail newJob = createJob();
         Trigger newTrigger = createTrigger();
 
-        boolean overWritePreviousJob = true;
+        if (newTrigger == null) {
+            return;
+        }
 
         try {
-            scheduler.addJob(newJob, overWritePreviousJob);
-
+            scheduler.addJob(newJob, true);
             scheduler.rescheduleJob(triggerKey, newTrigger);
         } catch (SchedulerException e) {
             LOGGER.info("Error with rescheduling of task.", e);
         }
-
     }
 
     private JobDetail createJob() {
-        return newJob().ofType(classObject)
+        return newJob().ofType(jobClass)
                 .withIdentity(jobKey)
                 .storeDurably()
                 .usingJobData(CommandJob.COMMAND_KEY, command)
                 .build();
     }
 
     private Trigger createTrigger() {
+        Trigger newTrigger = null;
+        if (CRON_STRING.equals(this.intervalType)) {
+            newTrigger = createCronTrigger();
+        } else if (SECOND_INTERVAL.equals(this.intervalType)) {
+            try {
+                Integer secondsInterval = Integer.parseInt(this.intervalString);
+                newTrigger = createSimpleTrigger(secondsInterval);
+            } catch (NumberFormatException e) {
+                LOGGER.warn("Unable to update Quartz job.  Invalid second interval specified : {}",
```

Comment:
I don't think this message would necessarily make sense to an admin if they were to see it.  They've just configured something called `Platform Command Scheduler` but now they are getting warnings about `Quartz`. Please update the log message to provide better context to the administrator.